test/Microsoft.ML.Tests/TrainerEstimators/SymSgdClassificationTests.cs

```diff
@@ -35,7 +37,7 @@ public void TestEstimatorSymSgdInitPredictor()
             var initPredictor = ML.BinaryClassification.Trainers.StochasticDualCoordinateAscent().Fit(transformedData);
             var data = initPredictor.Transform(transformedData);
 
-            var withInitPredictor = new SymSgdClassificationTrainer(Env, new SymSgdClassificationTrainer.Options()).Train(transformedData, initialPredictor: initPredictor.Model);
+            var withInitPredictor = new SymSgdClassificationTrainer(Env, new SymSgdClassificationTrainer.Options()).Train(transformedData, initialPredictor: initPredictor.Model as CalibratedPredictorBase<LinearBinaryModelParameters, PlattCalibrator>);
             var outInitData = withInitPredictor.Transform(transformedData);
```

Comment:
Can you split into two lines? It's could be hard to read on a smaller screen (like my laptop right now :) )

Reply:
Sure.

---
In reply to: [255170947](https://github.com/dotnet/machinelearning/pull/2435#discussion_r255170947) [](ancestors = 255170947)